api/resource.rb

```diff
@@ -94,9 +94,13 @@ module Properties
       # If you're writing a fine-grained resource (eg with nested_query) a value
       # must be set.
       attr_reader :identity
+
       # [Optional] (Api::Resource::NestedQuery) This is useful in case you need
       # to change the query made for GET requests only. In particular, this is
       # often used to extract an object from a parent object or a collection.
+      # Note that if both nested_query and custom_code.decoder are provided,
+      # the decoder should be included within the code handling the nested
```

Comment:
```suggestion
      # the decoder will be included within the code handling the nested
```

I don't think the MM author has much choice in the matter, right?

Reply:
true